cli.ts

```diff
@@ -68,6 +71,22 @@ function convertToStringArray(
   return arg;
 }
 
+export function upgrade(version: string) {
+  const versionWithoutVprefix = RegExp(/^\d\.\d\.\d/);
+  const redColor = '\u001b[31m'
+  if(version === 'latest') {
+    version = 'master'
+  }
+
+  if(versionWithoutVprefix.test(version)) {
+    console.warn(redColor,`Do you mean v${version}? If it's you should add v prefix`)
```

Comment:
Add a warn function to the `log.ts` file instead, also red is used for errors currently maybe orange would be a good alternative color. Also use the colors from the `std/fmt` module

Reply:
Good